config/cgimap.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=CGImap implementation of the OpenStreetMap API
+Documentation=https://wiki.openstreetmap.org/wiki/Cgimap
+After=network.target
+
+[Service]
+Type=forking
+ExecStart=/usr/local/bin/openstreetmap-cgimap --daemon
+PIDFile=/var/run/cgimap.pid
+
+[Install]
+WantedBy=multi-user.target
+Alias=cgimap
```

Comment:
Ubuntu 18.04 (latest used by Vagrant) now uses `systemd` instead of `init` so I created a basic service file for that just like `cgimap.conf` for `init`.

Reply:
I think this would be a topic for another pull request. I will skip this for now.